tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/static_method_tests.bal

```diff
@@ -218,7 +218,8 @@ function decimalParamAndReturn(decimal a1) returns decimal = @java:Method {
 } external;
 
  public function addTwoNumbersSlowAsync(int a, int b) returns int = @java:Method {
-    'class:"org/ballerinalang/nativeimpl/jvm/tests/StaticMethods"
+    'class:"org/ballerinalang/nativeimpl/jvm/tests/StaticMethods",
```

Comment:
Shall we write a test case for object attached interop function

Reply:
I think this scenario is still broken, added #26054 for this.